SignalServiceKit/src/Messages/Attachments/TSAttachment.m

```diff
@@ -180,6 +180,8 @@ - (NSString *)description {
         }
     } else if ([MIMETypeUtil isAnimated:self.contentType]) {
         return [NSString stringWithFormat:@"🎡 %@", attachmentString];
+    } else {
+        return [NSString stringWithFormat:@"📎 %@", attachmentString];
```

Comment:
@michaelkirk I meant to call out this change to you.  WDYT?  Also is 📻 the best emoji for sound/audio?